### PR TITLE
chore: Add marc1404 to OWNER_ALIASES

### PR DESCRIPTION
**Pull request by marc1404**
**How to categorize this PR?**

/area dev-productivity
/kind enhancement

**What this PR does / why we need it**:

As discussed with @rfranzke 

**Which issue(s) this PR fixes**:

_n.a._

**Special notes for your reviewer**:

_n.a._

**Release note**:

```other operator
NONE
```

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -14,6 +14,7 @@ aliases:
   - timebertt
   - timuthy
   - tobschli
+  - marc1404
```
Comment (acumino) [inline OWNERS_ALIASES]: Please add in the sorted order.
Welcome to the team @marc1404 🎉 . Indeed h2o is the way to go. 😄

**Review by acumino — CHANGES_REQUESTED** [review]

File: OWNERS_ALIASES (hunk 1 above)
Comment (marc1404) [reply OWNERS_ALIASES]: sorry for missing that, done 🔤 :)

**Review by rfranzke — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**